storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/StorIOSQLite.java

```diff
@@ -144,6 +145,110 @@
+     * so it's recommended to apply {@link Observable#observeOn(rx.Scheduler)}
+     * if you need to receive events on a special thread.
+     * <p/>
+     * Notice, that returned {@link Observable} is "Hot Observable", it never ends, which means,
+     * that you should manually unsubscribe from it to prevent memory leak.
+     * Also, it can cause BackPressure problems.
+     *
+     * @param tags set of tags that should be monitored.
+     * @return {@link rx.Observable} of {@link Changes} subscribed to changes of required tags.
+     */
+    @NonNull
+    public abstract Observable<Changes> observeChangesOfTags(@NonNull Set<String> tags);
+
+    /**
+     * Allows observer changes of required tag.
+     * <p/>
+     * Tags are the addition level of notifications that provides
+     * more detailed information about which particular changes were occurred.
+     * <p/>
+     * Notice that {@link StorIOSQLite} knows only about changes
+     * that happened as a result of Put or Delete Operations executed
+     * on this instance of {@link StorIOSQLite}.
+     * <p/>
+     * Emission may happen on any thread that performed Put or Delete operation,
+     * so it's recommended to apply {@link Observable#observeOn(rx.Scheduler)}
+     * if you need to receive events on a special thread.
+     * <p/>
+     * Notice, that returned {@link Observable} is "Hot Observable", it never ends, which means,
+     * that you should manually unsubscribe from it to prevent memory leak.
+     * Also, it can cause BackPressure problems.
+     *
+     * @param tag tag name to monitor.
+     * @return {@link rx.Observable} of {@link Changes} subscribed to changes of required tag.
+     */
+    @NonNull
+    public Observable<Changes> observeChangesOfTag(@NonNull String tag) {
+        checkNotNull(tag, "Tag can not be null or empty");
+        return observeChangesOfTags(Collections.singleton(tag));
+    }
+
+
+    /**
+     * Allows observe changes of required tables and tags.
+     * <p/>
+     * Notice that {@link StorIOSQLite} knows only about changes
+     * that happened as a result of Put or Delete Operations executed
+     * on this instance of {@link StorIOSQLite}.
+     * <p/>
+     * Emission may happen on any thread that performed Put or Delete operation,
+     * so it's recommended to apply {@link Observable#observeOn(rx.Scheduler)}
+     * if you need to receive events on a special thread.
+     * <p/>
+     * Notice, that returned {@link Observable} is "Hot Observable", it never ends, which means,
+     * that you should manually unsubscribe from it to prevent memory leak.
+     * Also, it can cause BackPressure problems.
+     *
+     * @param tags set of tags that should be monitored.
+     * @return {@link rx.Observable} of {@link Changes} subscribed to changes of required tags.
+     */
+    @NonNull
+    public abstract Observable<Changes> observeChangesOfTablesAndTags(
```

Comment:
+1 for removing it. It can get out of hands pretty quick if we need to introduce one more way of filtering changes. `observeChangesOfTablesAndTagsAndWhatever` plus combination look ugly.